src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/config/BackupProperties.java

```diff
@@ -16,6 +16,10 @@
     private String gcsBackupBucket;
     @NotNull(message = "gcs-timeout duration should be configured, but is not")
     private Duration gcsTimeout;
+    // This clears the entire cache on a time
     @NotNull(message = "gcs-flush duration should be configured, but is not")
     private long gcsFlushMilliseconds;
+    // Force flush stream after each write if true
+    @NotNull(message = "always-flush backup stream flushing behaviour should be specified as true or false")
```

Comment:
Value for `alwaysFlush` by default will be `false` as it is a primitive `boolean` type. So, you always will have this as either `true` or `false`. \@NotNull check will be `no-op`.

Reply:
oh right, i'll just take the notnull out. thanks!